_posts/2022-02-25-newpipe-0.22.1.md

```diff
@@ -0,0 +1,54 @@
+  > For those of you wondering, a `RecyclerView` is a special *view* component that allows showing many *items* of the same type in a list without overloading the system with too many views at the same time. It does so by binding only the items visible on the screen to the corresponding view component (for example a stream info item view), and when one such view is moved out on one side of the screen it is *recycled* to be used again once needed on the opposite side. The random crash that was fixed here was caused by a piece of code that *every now and then* tried to bind the same view component to multiple items at the same time, which is obviously not possible, hence the crash. Sorry for this out-of-section nerd talk, we just want to show you how hard it was to track this issue down.
+
+- [@Stypox](https://github.com/Stypox) (I, as the blog author, declare freedom from any conflict of interest) solved the graphical problem with the newly-introduced fast-forward/rewind arc not underlaying the system ui when playing in the fullscreen player. This was particularly visible when controls were hidden, since a non-animated border was shown around the animated arc.
+
+- Talking about system borders… users that have a phone with cutouts (you know, those little notches that phone manufactures really grow to love lately) might have noticed a misplaced player in the previous version. It was caused by an attempted improvement to watching videos in multi-window that evidently went wrong. This change, originally made by [@Cheechaii](https://github.com/Cheechaii), was reverted by [@litetex](https://github.com/litetex). Thanks to both!
+
+## Nerd Talk
+
+- The error catching library, ACRA, was updated by the master of libraries, [@TacoTheDank](https://github.com/TacoTheDank). Thank you!
+
+- [@litetex](https://github.com/litetex) improved the image minimizer GitHub action they introduced a couple of weeks ago to reduce the image size of (phone) screenshots included in issue bodies and comments.
+
+- [@Stypox](https://github.com/Stypox) beat [@TobiGr](https://github.com/TobiGr) on time and got a grip on incrementing Android's `compileSdk` from 30 (Android 11) to 31 (Android 12). This will allow updating more libraries to their newest versions in the future.
+
+- [@Stypox](https://github.com/Stypox) refactored a couple of lines in the big (we are talking BIG. Really, way **too BIG**) `Player` class. They would also like to point out that the whole team [is](https://github.com/TeamNewPipe/NewPipe/issues/7673) [committed](https://github.com/TeamNewPipe/NewPipe/issues/7939) to improving the messed-up player, which likes to spit out nasty bugs every once in a while due to its complexity. In a very distant future, expressions such as "the player is a hot pile of garbage" and "the code is a spaghetti monster of doom" might finally disappear! This is another step towards that future.
```

Comment:
What about adding the self praise as a joke to one of the paragraphs here? Smth like `The blog post author would like to point out here what a great contributor Stypox is.`

Reply:
Ok, you convinced me, but I think it wouldn't fit too well here, so I will do it on the line discussed before